R/image-interact-opts.R

```diff
@@ -137,3 +148,8 @@ brushOpts <- function(id, fill = "#9cf", stroke = "#036",
     resetOnNew = resetOnNew
   )
 }
+
+getThematicOption <- function(...) {
+  if (system.file(package = "thematic") == "") return(NULL)
```

Comment:
We should make this `system.file` pattern a separate function, like `is_installed`.